bundle-workflow/Jenkinsfile

```diff
@@ -29,24 +25,20 @@ pipeline {
             }
         }
         stage('build') {
-            agent {
-                node {
-                    label 'Jenkins-Agent-al2-x64-m5xlarge'
-                }
-            }
-            stages {
+            parallel {
                 stage('build-snapshots') {
                     environment {
                         SNAPSHOT_REPO_URL = "https://aws.oss.sonatype.org/content/repositories/snapshots/"
                     }
                     agent {
-                        node {
-                            label 'Jenkins-Agent-al2-x64-m5xlarge'
+                        docker {
+                            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                            image 'opensearchstaging/ci-runner:al2-x64-arm64-jdk14-node10.24.1-20210924'
```

Comment:
With this change, it seems like we are hardcoding the version of java even more firmly, do you have thoughts on after this change how we would resolve https://github.com/opensearch-project/opensearch-build/issues/74?

Reply:
Looks like the plan is to have multiple images with different JDKs. This works for me, unless we want to build a single image with many different JDKs and be able to select which one is used by default.

Reply:
Exactly like @dblock says we are going to have images with multiple setups.
We can also have multiple Jenkinsfile as well.

Reply:
Wouldn't that be a maintainence nightmare for so many images with multiple combinations?
Maybe we should have something that would allow installation of matrix specific versions.

Reply:
You choose between convenience vs preserve integrity of the env across all platforms.
I like the approach of having multiple versions.
Welcome more opinions.